### DISCUSSION TITLE
Cannot load module: timecode

### DISCUSSION THREAD

**Discussion by Nutzrobba2** [FrancescoCeruti/linux-show-player#278] [Q&A] [answered]
Hi, 

I have this warning message everytime i try to launch linux-show-player. OLA is running.

Did i miss something ?

Thanks for your help, this software looks amazing !

**Comment by s0600204** [comment]
Hi,

We're going to need a little more information before we can help you:

* What version of LiSP are you using?
* What version of Linux are you using?
* What's the exact _full_ error/exception message given?
* How did you acquire OLA? (And if you built it yourself, did you pass the `--enable-python-libs` argument?)

**Reply by Nutzrobba2** [reply]
Hi, thanks for the reply,


> What version of LiSP are you using?

0.5.3

> What version of Linux are you using?

Pop!_OS 22.04 LTS

> What's the exact full error/exception message given?

Hmm i just discovered the debug option for Lisp, i think the answer is here !

I am beginner with linux and python ( and with english also ! ), maybe i am missing the python plugin for ola ? I'll try it.


ImportError: OLA module not found, plugin not loaded.


> How did you acquire OLA? (And if you built it yourself, did you pass the --enable-python-libs argument?)

Installed via apt, version 0.10.8

Thx for your help

**Comment by Nutzrobba2** [comment, marked as answer]
Solved by installing ola-python...

So simple !

Thx
